.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 16.14.2
```

Comment:
if the purpose of having this file is just for the node version maybe we should use `.nvmrc` file instead, what do you think? 🤔

Reply:
@carryall I think the `.tool-versions` is used by asdf. Then he added it to support that tool.
I think we can keep both of them to support everyone who uses either asdf or nvm.

Reply:
Nice catch @carryall , I forgot `nvm` does not simply read the version from the `package.json` engine blocks.

+1 Hoang, `.tool-versions` is for asdf only, totally agree with the idea to support both! 👍 

`.nvmrc` file added in 49c7b78 ;-)